server/models/user.js

```diff
@@ -24,12 +48,40 @@ module.exports = function(sequelize, DataTypes) {
           callback(res);
         });
       },
-      toJson: function() {
+      toJson: function(currentUserId) {
```

Comment:
@dwoznicka http://docs.sequelizejs.com/en/latest/docs/scopes/ we can call scopes with parameters :+1: